Core/clim-basic/text-formatting.lisp

```diff
@@ -0,0 +1,130 @@
+;;; ---------------------------------------------------------------------------
+;;;     Title: Page layout abstraction
+;;;   Created: 2019-02-06 19:42
+;;;    Author: Daniel Kochmański <daniel@turtleware.eu>
+;;;   License: LGPL-2.1+ (See file 'Copyright' for details).
+;;; ---------------------------------------------------------------------------
+;;;
+;;;  (c) copyright 2002 by Alexey Dejneka <adejneka@comail.ru>
+;;;  (c) copyright 2019 by Daniel Kochmański <daniel@turtleware.eu>
+;;;
+;;; ---------------------------------------------------------------------------
+;;;
+;;; Page layout may has numerous properties which arrange things on a
+;;; stream with having a broader picture in mind. Text on a page may
+;;; have alignment and direction, margins, columns, paragraph settings
+;;; and much more. This file is a backon abstraction which may be used
```

Comment:
Is the word `backon` a typo? I don't know this word.

Reply:
beacon° I'll fix it